src/main/app-updater.ts

```diff
@@ -26,7 +27,7 @@ function handleAutoUpdateBackChannel(event: Electron.IpcMainEvent, ...[arg]: Upd
  * starts the automatic update checking
  * @param interval milliseconds between interval to check on, defaults to 24h
  */
-export function startUpdateChecking(interval = 1000 * 60 * 60 * 24): void {
+export const startUpdateChecking = once(function (interval = 1000 * 60 * 60 * 24): void {
```

Comment:
Does this fix an unrelated issue? Or just refactoring while accommodating protocol handling?

Reply:
It was fixing an issue due to the changes related to `src/main/index.ts:158`. Namely, if `"renderer:loaded"` gets called multiple times `startUpdateChecking` shouldn't be called on the subsequent events.

Reply:
I figured, but is it still a bug otherwise? If so, shouldn't it really be in its own PR?

Reply:
I figured that I should fix it in this PR since it was in this PR that I changed how the `renderer` lets `main` know it has been loaded.